src/module/actor/sheet/base.js

```diff
@@ -1010,58 +1031,41 @@ export class ActorSheetSFRPG extends ActorSheet {
                 ui.notifications.warn(game.i18n.format("SFRPG.ActorSheet.Inventory.Interface.ItemCollectionPickupNoGMError"));
             }
             return;
-        } else if (parsedDragData.pack) {
-            const pack = game.packs.get(parsedDragData.pack);
-            const itemData = await pack.getDocument(parsedDragData.id);
-
-            if (!this.acceptedItemTypes.includes(itemData.type)) {
-                // Reject item
-                ui.notifications.error(game.i18n.format("SFRPG.InvalidItem", { name: SFRPG.itemTypes[itemData.type], target: SFRPG.actorTypes[this.actor.type] }));
-                return;
-            }
-
-            if (itemData.type === "class") {
-                const existingClass = targetActor.findItem(x => x.type === "class" && x.name === itemData.name);
-                if (existingClass) {
-                    const levelUpdate = {};
-                    levelUpdate["data.levels"] = existingClass.data.data.levels + 1;
-                    existingClass.update(levelUpdate)
-                    return existingClass;
-                }
-            }
+        } else if (parsedDragData.uuid.includes("Compendium") || parsedDragData.pack) {
 
             const createResult = await targetActor.createItem(itemData.data._source);
             const addedItem = targetActor.getItem(createResult[0].id);
-            
-            if (game.settings.get('sfrpg','scalingCantrips') && addedItem.type === "spell") {
+
+            if (game.settings.get('sfrpg', 'scalingCantrips') && addedItem.type === "spell") {
                 _onScalingCantripDrop(addedItem, targetActor);
             }
-                
+
             if (!(addedItem.type in SFRPG.containableTypes)) {
                 targetContainer = null;
             }
-            
+
             const itemInTargetActor = await moveItemBetweenActorsAsync(targetActor, addedItem, targetActor, targetContainer);
             if (itemInTargetActor === addedItem) {
                 await this._onSortItem(event, itemInTargetActor.data);
                 return itemInTargetActor;
             }
 
             return itemInTargetActor;
-        } else if (parsedDragData.data) {
-            const sourceActor = new ActorItemHelper(parsedDragData.actorId, parsedDragData.tokenId, parsedDragData.sceneId);
+        } else if (parsedDragData.uuid.includes("Actor") || parsedDragData.data) {
```

Comment:
`uuid` isn't a property in the drag data on v9, so this throws an error when you drag items onto an actor. a `?` after `uuid` should fix this.